src/app/pages/test-centre-journal/components/view-journals-card/view-journals-card.ts

```diff
@@ -28,6 +28,9 @@ export class ViewJournalsCardComponent {
   @Input()
   testCentreResults: TestCentreDetailResponse;
 
+  @Input()
```

Comment:
We don't need this as an input, we're in the view-journals-card component, can just be defaulted to true - or pass true straight into createSlots

Reply:
removed